src/test/java/org/dcsa/api_validator/bkg/v1/bookings/GetBookingTest.java

```diff
@@ -0,0 +1,69 @@
+package org.dcsa.api_validator.bkg.v1.bookings;
+
+import org.apache.http.HttpStatus;
+import org.dcsa.api_validator.conf.Configuration;
+import org.testng.annotations.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.dcsa.api_validator.bkg.v1.BookingTestConfiguration.BOOKING_PATH;
+import static org.dcsa.api_validator.bkg.v1.BookingTestConfiguration.JSON_SCHEMA_VALIDATOR;
+
+public class GetBookingTest {
+
+	@Test
+	public void testGetValidBookingTest() {
+
+		given()
+			.auth()
+			.oauth2(Configuration.accessToken)
+		.when()
+			.pathParam("carrierBookingRequestReference", "ef223019-ff16-4870-be69-9dbaaaae9b11")
```

Comment:
We should consider making our tests less dependent on the test data?

Reply:
changed the tests to be more dynamic